### ISSUE TITLE
Error if the cwd is not the root of the package that is transpiled with babel

### ISSUE THREAD

**Issue by leMaik** [avocadowastaken/babel-plugin-direct-import#5]
This might be a very rare edge case, but I'd like to seet his fixed (a PR is on its way) and discuss this.

I have a react-styleguidist setup where I join multiple styleguides into one big styleguide. Each project (each individual style guide) uses babel and babel-plugin-direct-import. The problem is that when I build my big styleguide, babel does run from there and `process.cwd()` is not the root of every individual component anymore. This causes babel-plugin-direct-import not to find dependencies that are only used by individual components.

Long story short: `process.cwd()` not being the path of the module that uses babel-plugin-direct-import breaks it.

**Comment by avocadowastaken** [comment]
Good point, I was thinking about injecting cwd at the beginning (https://github.com/lodash/babel-plugin-lodash/blob/master/src/config.js#L17), but then decided to make it simpler.

**Comment by leMaik** [comment]
That might be a cleaner solution to my problem, though. :+1:

**Comment by leMaik** [comment]
Yeah, that would make the config syntax more complicated again and would require me to put the `cwd` into all `.babelrc` files.
The proposed PR doesn't require config changes. :-)

**Comment by avocadowastaken** [comment]
@leMaik yeah, I wrote it before I saw your PR , lets move our discussion there.
